src/main/resources/plugin.yml

```diff
@@ -27,18 +27,15 @@ commands:
   pyramid:
     description: (MultiCraft) builds a square pyramid at player position
     usage: /<command> <dimension> <material> [hollow 0:1]
-  rbuild:
-    description: (MultiCraft) initiates a region build
-    usage: /<command>
   rloc1:
     description: (MultiCraft) selects first corner of region
     usage: /<command>
   rloc2:
     description: (MultiCraft) selects second corner of region
     usage: /<command>
-  rrbuild:
+  rbuild:
     description: (MultiCraft) builds a structure from rloc1 to rloc2
-    usage: /<command> [material]
+    usage: /<command>
```

Comment:
I think it should be /<command> [material] since it accepts a material as an arg